src/lib/radio/radio-module.ts

```diff
@@ -6,15 +6,14 @@
  * found in the LICENSE file at https://angular.io/license
  */
 
-import {A11yModule} from '@angular/cdk/a11y';
 import {CommonModule} from '@angular/common';
 import {NgModule} from '@angular/core';
 import {MatCommonModule, MatRippleModule} from '@angular/material/core';
 import {MatRadioButton, MatRadioGroup} from './radio';
 
 
 @NgModule({
-  imports: [CommonModule, MatRippleModule, MatCommonModule, A11yModule],
```

Comment:
Is removing this one intentional? The radio button has a dependency on the focus monitor from a11y.

Reply:
Removing here is okay since radio doesn't use any of the a11y _directives_. However, this comment made me realize I shouldn't have remove `A11yModule` from the date picker module. Fixed that.